clang-tools-extra/clangd/tool/Check.cpp

```diff
@@ -367,7 +367,13 @@ class Checker {
     auto Hints = inlayHints(*AST, LineRange);
 
     for (const auto &Hint : Hints) {
-      vlog("  {0} {1} {2}", Hint.kind, Hint.position, Hint.label);
+      vlog("  {0} {1} [{2}]", Hint.kind, Hint.position, [&] {
+        return llvm::join(llvm::map_range(Hint.label,
+                                          [&](auto &L) {
+                                            return llvm::formatv("{{{0}}", L);
```

Comment:
There are three `{` in this format string, I assume that is a typo

Reply:
I admit this is quite strange, but the current `formatv` implementation doesn't require escaping `}`s (but does require doubling `{{`s for a literal `{`). See https://reviews.llvm.org/D83888.

Reply:
(I don't know why Sam's comment was not reflected in that patch at last -- maybe I can add that in a separate NFC patch.)